helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -212,11 +184,8 @@ data:
         [http.middlewares.auth-notebook.forwardauth]
           address = "http://{{ template "gateway.fullname" . }}-auth/?auth=notebook"
           trustForwardHeader = true
-    {{ if .Values.global.anonymousSessions.enabled }}
```

Comment:
If that header isn't included in the response from renku-notebooks, that doesn't matter, the code should not fail.